modules/auth/proc-adding-gitlab-oauth-as-an-authentication-provider.adoc

```diff
@@ -0,0 +1,61 @@
+[id='proc-adding-gitlab-oauth-as-an-authentication-provider_{context}']
+= Enabling the GitLab OAuth authentication provider
+
+{product} includes a GitLab authentication provider that can authenticate users by using GitLab OAuth.
+
+.Procedure
+. To allow {product-short} to authenticate with Gitlab, create an OAuth Application in Gitlab.
++
+Go to https://gitlab.com/-/profile/applications[GitLab User settings > Applications], and click the *Add new application* button.
+
+*Name*:: Enter your application name, such as `{product-short}`.
+*Redirect URI*:: Enter the backend authentication URI set in {product-short}:
+For local development::: `pass:[http://localhost:7007/api/auth/gitlab/handler/frame]`
+For a non-local deployment::: `pass:c,a,q[http://_<APP_FQDN>_/api/auth/gitlab/handler/frame]`
```

Comment:
@kadel Does it make sense to mention local development at all? I am deploying locally on OpenShift Local, and the URL is [backstage-developer-hub-rhdh-instance.apps-crc.testing/settings](https://backstage-developer-hub-rhdh-instance.apps-crc.testing/settings)

Reply:
local in backstage context means running locally on your laptop. 
But this is good, we are primarily targeting production deployments on Kubernetes or OpenShift clusters, customers won't be running rhdh locally on their laptops directly form git repo.

It should be ok to remove all local development references.